pkg/sysinfo/cgroup2_linux.go

```diff
@@ -0,0 +1,161 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
```

Comment:
The copyright boilerplate should be like
https://github.com/containerd/nerdctl/blob/1d9d22c4ce9fd9ada1c4e3fcd3143fd7caeb794e/pkg/strutil/strutil.go#L1-L29

Reply:
Thanks. Fixed.